schema/p2p_schema.go

```diff
@@ -62,7 +62,7 @@ func (P2PNetworkExtended) JSONSchemaOneOf() []interface{} {
 
 // VPN represents the vpn block in the Kairos configuration.
 type VPN struct {
-	Create bool          `json:"vpn,omitempty" default:"true"`
-	Use    bool          `json:"use,omitempty" default:"true"`
-	Envs   []interface{} `json:"env,omitempty"`
+	Create bool        `json:"vpn,omitempty" default:"true"`
+	Use    bool        `json:"use,omitempty" default:"true"`
+	Envs   interface{} `json:"env,omitempty"`
```

Comment:
Isn't it a map of string any?

Reply:
Or string string?

Reply:
Whoops merged without reading this. You might be right maybe it should be a map of string to interfaces